scripts/tools/generate_esp32_chip_factory_bin.py

```diff
@@ -534,6 +558,14 @@ def any_base_int(s): return int(s, 0)
     parser.add_argument('--pai-cert', help='The path to the PAI certificate in der format')
     parser.add_argument('--cd', help='The path to the certificate declaration der format')
 
+    # Options for esp_secure_cert_partition
+    parser.add_argument('--dac-in-secure-cert', action="store_true", required=False,
```

Comment:
required is false by default
```suggestion
    parser.add_argument('--dac-in-secure-cert', action="store_true",
```